conf/modules.config

```
@@ -344,34 +344,41 @@ process {
 
 // PREPARE_RECALIBRATION
 
-    withName: 'BASERECALIBRATOR|BASERECALIBRATOR_SPARK|GATHERBQSRREPORTS' {
+    withName: 'BASERECALIBRATOR|BASERECALIBRATOR_SPARK' {
         ext.prefix       = {"${meta.id}.recal"}
         publishDir       = [
             mode: params.publish_dir_mode,
             path: { "${params.outdir}/preprocessing/${meta.id}/recal_table" },
-            pattern: "*table"
+            pattern: "*table",
+            saveAs: { meta.num_intervals < 2 ? it : null }
```

```suggestion
            saveAs: { meta.num_intervals > 1 ? null : it }
```

You're using `meta.num_intervals >1` and `meta.num_intervals < 2`, why not using just one, might be easier to follow?

---

(I mean in the whole config file)

---

sure I think this stems from trial and error to avoid empty file creation

---

I know the feeling, and it's still a draft anyway